services/network_config_manager/src/main/resources/application.properties

```diff
@@ -24,6 +24,8 @@ dataplane.isovs=true
 grpc.min-threads = 100
 grpc.max-threads = 200
 grpc.threads-pool-name = grpc-thread-pool
+grpc.number-of-channels-per-host = 10
+grpc.number-of-warmups-per-channel = 1
```

Comment:
What would happen if admin misconfigures these two channel numbers, say 5 and 10 :-)

Reply:
I think, as long as they are positive numbers, then they should be good. 5 channels per host and 10 warmups in each channel could be desirable for some users. But anyway, maybe I need to check for 0 and negative numbers, let me add the code for that.

Reply:
Added sanity check for these two variables